extensions-core/hdfs-storage/pom.xml

```diff
@@ -245,6 +254,16 @@
               <groupId>com.sun.jersey</groupId>
               <artifactId>jersey-server</artifactId>
             </exclusion>
+            <!-- Following are excluded to remove security vulnerabilities: -->
+            <exclusion>
+              <!-- provided by core -->
```

Comment:
Is the `<!-- provided by core -->` here still accurate?

Reply:
It looks like both druid core and hdfs-storage were using version 1.9.4, I think it'd be more clear to move the security exclusion comment so that it only applies to `commons-beanutils-core`

Reply:
Removed